src/couch/src/couch_proc_manager.erl

```diff
@@ -321,7 +322,7 @@ find_proc(#client{lang = Lang, ddoc = DDoc, ddoc_key = DDocKey} = Client) ->
 
 find_proc(Lang, Fun) ->
     try iter_procs(Lang, Fun)
-    catch ?STACKTRACE(error, Reason, StackTrace)
+    catch error:Reason:StackTrace ->
```

Comment:
Is it intentional omission of
```
    catch error:Reason:StackTrace ->
        ?LOG_ERROR(#{
            what => process_lookup_failure,
            result => error,
            details => Reason,
            stack => StackTrace
        }),
```

Reply:
I removed all the ?STACKTRACE invocations because I was removing the macro, but I didn't add logging to these modules because as far as I can tell they are leftover cruft and should be removed entirely in favor of the ones in `couch_js`